.github/workflows/ci.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         ruby: ["2.5", "2.6", "2.7", "3.0", "3.1"]
+        redis-rb: ["< 5", "edge"]
```

Comment:
NB: I added this to make sure it doesn't only work on my machine, but it's up to you wether or not you want this in your CI.